pom.xml

```
@@ -272,7 +305,7 @@
 				<artifactId>maven-javadoc-plugin</artifactId>
 				<version>3.0.1</version>
 				<configuration>
-					<excludePackageNames>io.github.hapjava.impl</excludePackageNames>
+					<excludePackageNames>io.github.hapjava.server.impl</excludePackageNames>
```

```suggestion
					<excludePackageNames>io.github.hapjava.server.impl</excludePackageNames>
					<source>8</source>
```

needed to compile on JDK11+